ui/api/src/main/java/org/jboss/forge/addon/ui/input/SelectComponent.java

```diff
@@ -30,4 +31,9 @@
    Converter<VALUETYPE, String> getItemLabelConverter();
 
    IMPLTYPE setItemLabelConverter(Converter<VALUETYPE, String> converter);
+
+   Converter<VALUETYPE, String> getInteractionItemLabelConverter();
+
```

Comment:
If the purpose is work only in GUI. I agree. But, sometimes I want to set the convert for two case.

But, for this case.. The interaction mode shows a different presentation of the not interaction. And the user could set how he want to use.